README.md

```diff
@@ -2,8 +2,8 @@
 
 # ClearSky - A fast Minecraft PE server software based on [PocketMine-MP](https://github.com/pmmp/PocketMine-MP)
 ClearSky is a fast Minecraft: Pocket Edition server software, finally back as a real FORK of [PocketMine-MP](https://github.com/pmmp/PocketMine-MP)
-## Why should i use ClearSky?
-#### Because our awesome, highly skilled Team is mostly focusing on making the server behave like singleplayer, whilest having high performant and clean code.
+## Why should I use ClearSky?
+#### Because our awesome, highly skilled Team is mostly focusing on making the server behave like singleplayer, whilst having high performance and clean code.
```

Comment:
why should that be the "performance"? i meant that the code is performant, but both ways work i guess